views/js/smaily_module.js

```diff
@@ -93,6 +93,34 @@ $(document).ready(function() {
     });
   });
 
+  // Generate RSS product feed URL if options change.
+  $(".smaily-rss-options").change(function(event) {
+    var rss_url_base = smaiy_rss_url + '?';
+    var url_parameters = {};
+
+    var rss_limit = $('#SMAILY_RSS_LIMIT').val();
+    if (rss_limit != "") {
+      parameters.limit = rss_limit;
+    }
+
+    var rss_order_by = $('#SMAILY_RSS_ORDER_BY').val();
+    if (rss_order_by != "") {
+      parameters.order_by = rss_order_by;
+    }
+
+    var rss_order_way = $('#SMAILY_RSS_ORDER_WAY').val();
+    if (rss_order_way != "") {
+      parameters.order_way = rss_order_way;
+    }
+
+    var rss_category_id = $('#SMAILY_RSS_CATEGORY_ID').val();
+    if (rss_category != "all_products") {
+      parameters.id_category = rss_category_id;
+    }
+
+    $('#smaily-rss-feed-url').html(rss_url_base + $.param(url_parameters));
+  });
```

Comment:
Does not work. When changing a value in field I get an error:
> Uncaught ReferenceError: parameters is not defined
\<anonymous\> http://127.0.0.1:8080/modules/smailyforprestashop/views/js/smaily_module.js:108

Reply:
I renamed parameters to url_parameters but didn't pay close attention or test it.
Should work now.